interfaces/Portalicious/package-lock.json

```diff
@@ -13012,6 +11798,117 @@
         "node": ">=6"
       }
     },
+    "node_modules/knip": {
+      "version": "5.46.0",
+      "resolved": "https://registry.npmjs.org/knip/-/knip-5.46.0.tgz",
+      "integrity": "sha512-WedHSK5xNBWYgm64Rt5B9b0CVXL2kRBcyCeet3NHgdv9en3QE4AWSDPEiX48NoPUBW3h//9S0VwLF5MG/MPi3g==",
+      "dev": true,
+      "funding": [
+        {
+          "type": "github",
+          "url": "https://github.com/sponsors/webpro"
+        },
+        {
+          "type": "opencollective",
+          "url": "https://opencollective.com/knip"
+        },
+        {
+          "type": "polar",
+          "url": "https://polar.sh/webpro-nl"
+        }
+      ],
+      "license": "ISC",
+      "dependencies": {
+        "@nodelib/fs.walk": "3.0.1",
+        "@snyk/github-codeowners": "1.1.0",
+        "easy-table": "1.2.0",
+        "enhanced-resolve": "^5.18.0",
+        "fast-glob": "^3.3.3",
+        "jiti": "^2.4.2",
+        "js-yaml": "^4.1.0",
+        "minimist": "^1.2.8",
+        "picocolors": "^1.1.0",
+        "picomatch": "^4.0.1",
+        "pretty-ms": "^9.0.0",
+        "smol-toml": "^1.3.1",
+        "strip-json-comments": "5.0.1",
+        "summary": "2.1.0",
+        "zod": "^3.22.4",
+        "zod-validation-error": "^3.0.3"
+      },
```

Comment:
Nice that a tool that's purpose is to "remove unnecessary dependencies", has _this many_ dependencies... 🧐